modules/static/googlecloudplatform/bq-schema-api/rsync.incl

```diff
@@ -0,0 +1,3 @@
++ LICENSE
++ *.proto
```

Comment:
Are these really meant to be imported from the top-level? I would've expected them to be put in a directory matching the package name.

Reply:
[bq_field.proto](https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema/blob/31a7e43419f7c19d79de0bb506798bc602287e80/bq_field.proto)

[bq_table.proto](https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema/blob/31a7e43419f7c19d79de0bb506798bc602287e80/bq_table.proto)

Are both at the root of the repository.